Build/Scripts/runTests.sh

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env bash
+
+#
+# TYPO3 core test runner based on docker and docker-compose.
+#
+
+# Function to write a .env file in Build/testing-docker
+# This is read by docker-compose and vars defined here are
+# used in Build/testing-docker/docker-compose.yml
+setUpDockerComposeDotEnv() {
+    # Delete possibly existing local .env file if exists
+    [ -e .env ] && rm .env
+    # Set up a new .env file for docker-compose
+    {
+        echo "COMPOSE_PROJECT_NAME=local"
+        # To prevent access rights of files created by the testing, the docker image later
+        # runs with the same user that is currently executing the script. docker-compose can't
+        # use $UID directly itself since it is a shell variable and not an env variable, so
+        # we have to set it explicitly here.
+        echo "HOST_UID=`id -u`"
+        # Your local user
+        echo "ROOT_DIR=${ROOT_DIR}"
+        echo "HOST_USER=${USER}"
+        echo "DOCKER_PHP_IMAGE=${DOCKER_PHP_IMAGE}"
+        echo "SCRIPT_VERBOSE=${SCRIPT_VERBOSE}"
+        echo "CGLCHECK_DRY_RUN=${CGLCHECK_DRY_RUN}"
+    } > .env
+}
+
+# Load help text into $HELP
+read -r -d '' HELP <<EOF
+EXT:reference-coreapi test runner. Check code styles, lint PHP files and some other details.
+
+Recommended docker version is >=20.10 for xdebug break pointing to work reliably, and
+a recent docker-compose (tested >=1.21.2) is needed.
+
+Usage: $0 [options] [file]
+
+No arguments: Run all unit tests with PHP 8.1
+
+Options:
+    -s <...>
+        Specifies which test suite to run
+            - cgl: cgl test and fix all php files
+            - composerUpdate: "composer update", handy if host has no PHP
+            - lint: PHP linting
+            - rector: Apply Rector rules
+
+    -p <8.1|8.2>
+        Specifies the PHP minor version to be used
+            - 8.1 (default): use PHP 8.1
+            - 8.2: use PHP 8.2
+
+    -u
+        Update existing typo3/core-testing-*:latest docker images. Maintenance call to docker pull latest
+        versions of the main php images. The images are updated once in a while and only the youngest
+        ones are supported by core testing. Use this if weird test errors occur. Also removes obsolete
+        image versions of typo3/core-testing-*.
+
+    -v
+        Enable verbose script output. Shows variables and docker commands.
+
+    -h
+        Show this help.
+
+Examples:
+    # Run unit tests using PHP 8.1
```

Comment:
```suggestion
    # Run checks using PHP 8.1
```

As we have no unit tests here.